etc/scripts/json2csv.py

```diff
@@ -114,9 +114,6 @@ def collect_keys(mapping, key_group):
             if not path.endswith('/'):
                 path = path + '/'
 
-        # alway create a root directory
-        path = '/code' + path
```

Comment:
@agneet42 You would have to add in a new test to make sure the option works correctly, that it prefixes `/code` when the option is selected.